docs/historical/cri/proposal.md

```diff
@@ -109,3 +109,4 @@ Following items are out of the scope of this design, we may address them in futu
 |  Runtime Management Integration  |          |          |     ✓    |     ✓    |     ✓    |     ✓    |
 |   Image Management Integration   |          |          |          |     ✓    |     ✓    |     ✓    |
 | Container Networking Integration |          |          |          |          |     ✓    |     ✓    |
```

Comment:
project checker is complaining about the extra line?

Reply:
Noted and corrected. Thank you very much.